notebooks/00_quick_start/wide_deep_model_movielens.ipynb

```diff
@@ -0,0 +1,1388 @@
+{
```

Comment:
<p>As before, this is too verbose</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>Please see my reply to the previous comment</p>